pack/df.json

```diff
@@ -6,7 +6,7 @@
     "faction_code": "anarch",
     "faction_cost": 1,
     "flavor": "With each passing day alone, Hoshiko found it harder to think. With each hour, the static grew louder.",
-    "illustrator": "Photo Tammy Gann/Unsplash. Deep Dream",
+    "illustrator": "Tammy Gann/Unsplash, Deep Dream",
```

Comment:
Maybe reach out to Nisei creative for this one. The others make sense as either FFG or Nisei worked with both parties, but these deep dream ones might have different rules about attribution.

Reply:
yeah, i asked.  Folks weren't clear, so i'm changing this to be treated as a single illustrator.